README.md

```diff
@@ -33,10 +38,10 @@ a `formatted` property that looks like this: `Mountain View, CA, United States,
 
 The utility geoip-gen reads csv files provided from GeoLite, and turns them into a 32-way branching b-tree, which is stored as ipfs json objects.
 
-There is a generator included, that can be called like this:
+There is a generator included, that can be run with
 
 ```bash
-$ node geoip-gen.js path/GeoLite-Blocks.csv path/GeoLite-Location.csv
```

Comment:
shall we add this script as a bin script to be installed and accessible globally? :)